### DISCUSSION TITLE
Mensaje de error sugiere que la biblioteca de etiquetas PrimeFaces no puede encontrar la definición de la etiqueta layout en el espacio de nombres http://primefaces.org/ui.

### DISCUSSION THREAD

**Discussion by rortegato** [primefaces/community#878] [PrimeFaces Templates] [answered]
Estoy actualizando de primefaces version 7 a la 10, cuando ejecuto me muestra este error: javax.faces.view.facelets.TagException: /login.xhtml @22,27 <p:layout> Tag Library supports namespace: http://primefaces.org/ui, but no tag was defined for name: layout. 
El mensaje de error sugiere que la biblioteca de etiquetas PrimeFaces no puede encontrar la definición de la etiqueta layout en el espacio de nombres http://primefaces.org/ui.
Tengo la version del primefaces-extensions 8. Alguien podria ayudar.

**Comment by mertsincan** [comment, marked as answer]
Hi, 
This component has already removed in 10+. Please see; https://github.com/primefaces/primefaces/issues/2168

Best Regards,
